### PR TITLE
Add detailed location history

### PR DESCRIPTION
**Pull request by chris-sophos**
Update 1.4 states that it adds 4 new locations, these are likely 4 of London, Edinburgh, Copenhagen, Gothenburg, Grimsby. If it's possible to figure out which 4, it may be beneficial to add that info to this list

### CODE REVIEW

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
7. Budapest [[1.14.1](https://www.ipa4fun.com/ipa/267007/)]
```

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
8. Florence [[19.0](https://www.ipa4fun.com/ipa/578209/)]
```
Updating since it seems they had a version number jump.

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
9. Porto [[20.1](https://www.ipa4fun.com/ipa/625102/)]
```

File: README.md
Comment (ekohilas) [inline README.md]: Thanks for correcting this! I think this would make more grammatical sense now.
```suggestion
A lot of the earlier details such as what order which locations were unlocked or released in are no longer easily found without restarting the game, however looking at the game's [version history][history], the determined is:
```

File: README.md
Comment (ekohilas) [reply README.md]: I'm not sure if `ipa4fun` hasn't stored `v1.14`, or if it never existed, but in either case it's probably more accurate to put this since we have a source.

File: README.md
Comment (ekohilas) [inline README.md:173]: We should probably update this to be 30, but we can leave that for a different PR.

File: README.md
Comment (ekohilas) [inline README.md]: > Update 1.4 states that it adds 4 new locations, these are likely 4 of London, Edinburgh, Copenhagen, Gothenburg, Grimsby. If it's possible to figure out which 4, it may be beneficial to add that info to this list

I don't remember how unlocking cities works in the game, but it may be possible to assert this assumption based on the unlocking order. Either way, I think it's a fair assumption for us to make, so long as we specify that we're making that assumption.

```suggestion
2. "UK & Scandinavia" (Assumed to be Copenhagen, Edinburgh, Gothenburg, Grimsby,  London) [[1.4](https://www.ipa4fun.com/ipa/68834/)]
```

I'm also replacing the `()` with `[]` so that the versioning is different from the annotation.

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
1. Prague & Warsaw [[v1.3](https://www.ipa4fun.com/ipa/64334/)]
```

Adding `v` since the version numbers seems to break Github's markdown rendering of footnotes.

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
3. Bernina [[1.6](https://www.ipa4fun.com/ipa/81402/)]
```

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
4. Belfast [[1.8](https://www.ipa4fun.com/ipa/92847/)]
```

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
5. Mostar [[1.10](https://www.ipa4fun.com/ipa/106999/)]
```

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
6. Madrid [[1.12](https://www.ipa4fun.com/ipa/124229/)]
```

File: README.md
Comment (ekohilas) [inline README.md:183]:
```suggestion
7. Budapest [[1.14](https://www.ipa4fun.com/ipa/267007/)]
```

File: README.md
Comment (ekohilas) [inline README.md:184]:
```suggestion
8. Florence [[1.19](https://www.ipa4fun.com/ipa/578209/)]
```

**Review by ekohilas — COMMENTED** [review]
Wow thanks for this!
It seems that the descriptions in the `ipa4fun` version history page is broken, but you were able to find the actual notes after clicking in?

I've left some suggested changes for accuracy, and after that I think it looks good to merge in!

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
3. Bernina [[v1.6](https://www.ipa4fun.com/ipa/81402/)]
```

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
4. Belfast [[v1.8](https://www.ipa4fun.com/ipa/92847/)]
```

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
5. Mostar [[v1.10](https://www.ipa4fun.com/ipa/106999/)]
```

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
6. Madrid [[v1.12](https://www.ipa4fun.com/ipa/124229/)]
```

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
7. Budapest [[v1.14.1](https://www.ipa4fun.com/ipa/267007/)]
```

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
8. Florence [[v19.0](https://www.ipa4fun.com/ipa/578209/)]
```

File: README.md
Comment (ekohilas) [inline README.md]:
```suggestion
9. Porto [[v20.1](https://www.ipa4fun.com/ipa/625102/)]
```

**Review by ekohilas — APPROVED** [review]
oops forgot the v's

**Comment by ekohilas** [comment]
The visual diff is still weird, but it's probably just a bug with Github.
Let me know if there's anything else I missed!
(Also I think you should be able to squash and merge after approval?

File: README.md
Comment (chris-sophos) [reply README.md:173]: Yeah, I've done that (and bumped the date) in a different PR, I meant to also add Porto to the list there but got carried away

**Comment by chris-sophos** [comment]
Seems like you've set a rule that only users with write access can merge pull requests, so you'll have to merge it yourself I suppose

**Comment by ekohilas** [comment]
> Seems like you've set a rule that only users with write access can merge pull requests, so you'll have to merge it yourself I suppose

Thanks for letting me know!
That seems to be a default rule that I couldn't find how to turn off.
I'll merge it for now and then lookup how to disable.

**Comment by chris-sophos** [comment]
Unresolved convo lol. Oh well

**Pull request merged**
